tests/models/gemma/test_modeling_gemma.py

```diff
@@ -500,6 +467,7 @@ def test_sdpa_equivalence(self):
     @require_flash_attn
     @require_torch_gpu
     @pytest.mark.flash_attn_test
+    @is_flaky
```

Comment:
For the comment [here](https://github.com/huggingface/transformers/pull/29805#discussion_r1535568737), [my response](https://github.com/huggingface/transformers/pull/29805#discussion_r1540782047) is

> I will open an issue but leave @younesbelkada to fill more details on the issue page.